scheduler/src/cook/queries.clj

```diff
@@ -28,3 +28,8 @@
             [?e :resource.type/mesos-name ?ident]]
           db)
        (map first)))
+
+(defn get-quota-resource-types
+  "Return a list of resource types used for quota, :cpus :mem :gpus ..."
```

Comment:
E.g., 'quota' is an overloaded term.We should update comments to indicate what is meant, 'e.g., these resource types are used by fenzo' or 'these are used to filter the queue' or 'these resource types are used for both'. (Per our phone call, this is being refactored, but when you do that, keep an eye out for analogous ambiguity.